src/cmd/root.go

```diff
@@ -177,6 +178,13 @@ func rootHelp(cmd *cobra.Command, args []string) {
 }
 
 func rootRun(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		rootFlags.assumeYes = true
```

Comment:
Umm... we shouldn't set `assumeYes` to `true` in this code path, because it can be surprising. eg., someone who has just run into Toolbox and casually types in `toolbox` will suddenly be downloading 100s of MBs without having a way to bow out of it.

I don't think this is necessary for compatibility with `coreos/toolbox`.

Reply:
The goal was to be compatible with `coreos/toolbox`. Seems like that was unneeded.